applications/tari_indexer/proto/dan/common.proto

```diff
@@ -0,0 +1,15 @@
+// Copyright 2022 The Tari Project
+// SPDX-License-Identifier: BSD-3-Clause
+
+syntax = "proto3";
```

Comment:
Let's rather move the proto files to a single `tari_dan_app_grpc` crate like we do in the base layer